### ISSUE TITLE
Feature Request: Don't use "@import" (Use "@use" instead)

### ISSUE THREAD

**Issue by sharshuv-quotient** [kristerkari/stylelint-scss#530]
Hey,
I want a rule that forbid using "@import", and allows only "@use" and "@forward" but couldn't find one.
Any help?

**Comment by kristerkari** [comment]
I'm not sure what you are after exactly, but you could always simply blaclist `@import` and suggest to use `@use` and `@forward`:

```js
    "at-rule-disallowed-list": [
      [
        "import",
      ],
      {
        severity: "error",
        message: "Use @use and @forward instead."
      }
    ]
```
https://github.com/stylelint/stylelint/tree/master/lib/rules/at-rule-disallowed-list

**Comment by blg002** [comment]
@kristerkari  I believe what he's after is the [Sass teams deprecation of the `@import`](https://sass-lang.com/documentation/at-rules/import) rule and recommendation to use `@use` and `@forward` instead.

> The Sass team discourages the continued use of the @import rule. Sass will gradually phase it out over the next few years, and eventually remove it from the language entirely. Prefer the @use rule instead.

The example you provided is a good work around but i think this change will eventually need to be reflecting throughout. For example `dollar-variable-first-in-block`'s `ignore` options

```
"dollar-variable-first-in-block": [
  true,
  { "ignore": ["comments", "imports", "**use**", "**forward**"] }
]

**Comment by kristerkari** [comment]
Yeah, it's true that each rule that is affected needs to be updated accordingly, but what comes to @sharshuv-quotient 's question, there is no need to add a new rule for something that can be done in stylelint already.
